source/index.html

```diff
@@ -1,12 +1,211 @@
+                  </svg>  
+                </a>
+              </li>
+              <li class="footer-social__item">
+                <a class="footer-social__link">
+                  <svg class="footer-social__icon--telegram" width="20" height="18" viewBox="0 0 20 18" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path fill-rule="evenodd" clip-rule="evenodd" d="M19.4608 0.152905C19.6393 0.302216 19.7255 0.534898 19.6872 0.764481L17.0611 16.5214C17.0244 16.7411 16.8789 16.9273 16.6745 17.0159C16.47 17.1045 16.2346 17.0833 16.0493 16.9597L8.17082 11.7074C8.01771 11.6054 7.91447 11.4437 7.88624 11.2619C7.858 11.08 7.90738 10.8947 8.02233 10.751L10.633 7.48773L6.27303 10.3944C6.077 10.525 5.82597 10.5407 5.61524 10.4353L0.362931 7.80915C0.127693 7.69153 -0.0146529 7.44488 0.00119976 7.18235C0.0170525 6.91983 0.188044 6.6921 0.435726 6.60364L18.8188 0.0382579C19.038 -0.0400244 19.2822 0.00359465 19.4608 0.152905ZM2.32626 7.32275L5.86168 9.09046L13.4231 4.0495C13.6925 3.86994 14.0525 3.91349 14.2712 4.1521C14.49 4.3907 14.5022 4.75314 14.3 5.00591L9.49502 11.0121L15.9328 15.3039L18.2083 1.65058L2.32626 7.32275Z" fill="#666666"/>
+                  </svg>    
+                </a>
+              </li>
+            </ul>
+
+            <div class="academy-logo">
+              <a class="academy-logo__link">
+                <span class="academy-logo__title">HTML Academy</span>
+                <svg class="academy-logo__image"width="27" height="34" viewBox="0 0 27 34" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Тут должен быть пробел
```suggestion
                <svg class="academy-logo__image" width="27" height="34" viewBox="0 0 27 34" fill="none" xmlns="http://www.w3.org/2000/svg">
```